courses/machine_learning/deepdive2/recommendation_systems/solutions/1_als_bqml.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Oh nice! You fixed a bug where students could accidentally add duplicate data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Whoops, noticed that line 14 and 15 have 3 spaces instead of 4.</p>